src/modules/AutoinstPartition.rb

```diff
@@ -160,7 +175,7 @@ def getPartitionDescription(p, enableHTML)
       part_desc = Builtins.sformat(
         "%1,%2",
         part_desc,
-        Partitions.FsIdToString(Ops.get_integer(p, "partition_id", 131))
+        Y2Storage::PartitionId.new_from_legacy(p.fetch("partition_id", 131)).to_s
```

Comment:
NP: magic number (131) in the code. Worse, in decimal which doesn not tell much to anyone, unlike 0x83 which many admins/users know as "partition type Linux".

Agreed, this was in the old code, too. Still not pretty ;-)